test/e2e/tests/send-eth.spec.js

```diff
@@ -375,13 +378,21 @@ describe('Send ETH from dapp using advanced gas controls', function () {
           '[data-testid="transaction-list-item-primary-currency"]',
         );
         await txValue.click();
-        const baseFeeValue = await driver.waitForSelector(
-          {
-            text: '0.000000025',
-          },
-          { timeout: 15000 },
+        await driver.waitForSelector({
+          xpath: "//div[contains(text(), 'Base fee')]",
+        });
+
+        const allFeeValues = await driver.findElements(
+          '.currency-display-component__text',
         );
-        assert.equal(await baseFeeValue.getText(), '0.000000025');
+
+        // In my tests, I've some of these values change between e2e runs.
+        // I think it might be better to just check if it's there and numeric
+        assert.equal(allFeeValues.length, 6);
```

Comment:
```
Below lines check that fee values are numeric.

The reason is that in my tests, I've seen some of these values change between e2e runs.
I think it might be better to just check if it's there and numeric
```